code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -12,7 +12,7 @@
 	var/eatverb
 	var/wrapped = 0
```

Comment:
correct me if I am wrong, but I think this variable is unused.

Reply:
Looks like eatverb isn't used either...